py/desispec/qa/qa_quicklook.py

```diff
@@ -214,6 +217,84 @@ def integrate_spec(wave,flux):
     integral=np.trapz(flux,wave)
     return integral
 
+def sky_resid(param, frame, skymodel, quick_look=False):
```

Comment:
Why move this method inside qa_quicklook?

If isn't specific to quicklook and I would argue that
having it within the desispec.sky module makes it
easier to understand what is being used.